### ISSUE TITLE
catch this in FeatureDefinition validation

### ISSUE THREAD

**Issue by isaacs** [tierrun/node-sdk#15]
```
failed: stripe: req_zLSbxY6FtxGAct type:invalid_request_error param:tiers message:You may only specify one of these parameters: tiers, transform_quantity.: feature:addvehicles@plan:ad@5
```

Also, take note of this in Model Builder's support for quantity transforms

**Comment by isaacs** [comment]
Can have a *single* tier, that's fine, or a flat rate, but not *multiple* tiers.

**Comment by isaacs** [comment]
Fixed on b3a2be3
